lib/stat_tracker.rb

```diff
@@ -69,4 +70,19 @@ def average_goals_by_season
       goals_by_season
     end
   end
+
+  def percentage_home_wins
+    home_wins = @game_collection.games.find_all {|game| game.home_goals > game.away_goals}
+    home_wins.count.to_f / (@game_collection.games.count.to_f).round(2)
+  end
+
+  def percentage_visitor_wins
+    home_wins = @game_collection.games.find_all {|game| game.home_goals < game.away_goals}
+    home_wins.count.to_f / (@game_collection.games.count.to_f).round(2)
+  end
+
+  def percentage_ties
+    tied_games = @game_collection.games.find_all {|game| game.home_goals == game.away_goals}
+    tied_games.count.to_f / (@game_collection.games.count.to_f).round(2)
```

Comment:
I think we should be using #length or @size for these, count is an enumerable and will soak up processing power.

Reply:
Yes, I was thinking about using #count, I will change that now.